### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_ Url Rewrite-Server Variables_Keyboard: After add button focus is going to the ok button instead of going to the textfield

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#197]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #Keyboard; #Chrome; #MAS2.4.3

**Environment Details:**
URL: https://manage.iis.net/webserver/url-rewrite
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Keyboard
MAS ID Violated: MAS 2.4.3

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Url Rewrite and press enter
6. Tab till Server variables tab and press enter
7. Tab till add button and press enter. Verify focus order is proper or not after add button

**Actual Result:**
After add button focus is going to ok button instead of going to the text field

**Expected Result:**
After add button focus should go to the text field instead of going to the ok button

**Note:**
This issue has been observed in all the pages which has similar controls

**User Impact:**
Disable users may find difficult while working with the application if focus order is not proper

**Attachments:**
[Bug197_server variables_focus order not proper after add button.pptx](https://github.com/Microsoft/IIS.WebManager/files/2336126/Bug197_server.variables_focus.order.not.proper.after.add.button.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.iis.net%2F&data=05%7C01%7Cv-bhaprakash%40microsoft.com%7C7a737faa4af14c4ad31a08db634ab100%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638212945750407456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Q7lvppcGrDLIRxk6EaLiumHk2i6E5S784T%2BniobLEu4%3D&reserved=0) home page will be removed.
